src/components/Tracker.vue

```diff
@@ -3,21 +3,54 @@
     <fullscreen v-model="fullscreen">
       <div class="d-flex flex-column h-100">
         <div style="flex: 1 1 auto">
-          <div id="map" class="w-100" style="height: 75vh" :class="{'h-100': fullscreen}"></div>
-          <div id="serverStatus" class="position-absolute">
+          <div
+            id="map"
+            class="w-100"
+            style="height: 75vh"
+            :class="{ 'h-100': fullscreen }"
+          ></div>
+          <div v-if="!fullscreen" id="serverStatus" class="position-absolute">
             <Status></Status>
-            <b-badge v-if="showFullScreen" v-b-tooltip.hover :title="FullscreenDesc" role="button" variant="primary"
-                     @click="toggleFullscreen">
+            <b-badge
+              v-if="showFullScreen"
+              v-b-tooltip.hover
+              :title="FullscreenDesc"
+              role="button"
+              variant="primary"
+              @click="toggleFullscreen"
+            >
               <BIconFullscreen v-if="!fullscreen"></BIconFullscreen>
               <BIconFullscreenExit v-if="fullscreen"></BIconFullscreenExit>
-              {{ fullscreen ? 'Exit' : 'Enter' }} Fullscreen
+              {{ fullscreen ? "Exit" : "Enter" }} Fullscreen
             </b-badge>
           </div>
-          <div id="schedule" class="position-absolute">
-             <Fullscrn_Schedule v-if="fullscreen"></Fullscrn_Schedule>
-          </div>
-          <div id="qrcode" class="position-absolute">
-            <Fullscrn_qrcode v-if="fullscreen"></Fullscrn_qrcode>
+          <div v-if="fullscreen" id="sidebar" class="position-absolute">
+            <div v-if="fullscreen" id="serverStatus">
```

Comment:
Duplicate id

Reply:
should be fixed in 5c49b8b72e9c565a1273038ad59a50dd63f57b0e. If youre refering to duplicate id of "serverStatus".